concepts/match-basics/about.md

```diff
@@ -138,11 +176,36 @@ fn toss(value: u8) {
 }
 
 fn main() {
-let x: u8 = 2;
-let y: u8 = 3;
-let z: u8 = 5;
-toss(x); // prints "you won!".
-toss(y); // prints "you can roll again!".
-toss(z); // prints "you lost...".
+    let x: u8 = 2;
+    let y: u8 = 3;
+    let z: u8 = 5;
+
+    toss(x); // prints "you won!".
+    toss(y); // prints "you can roll again!".
+    toss(z); // prints "you lost...".
 }
 ```
+
+## Matching Tuples
+
+Tuples can also be matched against a set of patterns. These patterns are designed similarly as a tuple.
+
+```rust
+...
+#[derive(Drop)]
+enum DayType {
+    Week,
+    Weekend,
+    Holiday
+}
+
+fn vending_machine_accept(c: (DayType, Color)) -> bool {
```

Comment:
This example makes little logical sense: why would a vending machine accept a _color_. Either create a different example that makes sense with `Color`, or copy the `Coin` enum from TCB into this example and use that instead.